test/test_prototype_datasets_utils.py

```diff
@@ -0,0 +1,46 @@
+import sys
+
+import numpy as np
+import pytest
+import torch
+from datasets_utils import make_fake_flo_file
+from torchvision.datasets._optical_flow import _read_flo as read_flo_ref
+from torchvision.prototype.datasets.utils._internal import read_flo, fromfile
+
+
+@pytest.mark.filterwarnings("error:The given NumPy array is not writeable:UserWarning")
+@pytest.mark.parametrize("mode", ("rb", "r+b"))
+@pytest.mark.parametrize(
+    ("np_dtype", "torch_dtype", "byte_order"),
```

Comment:
maybe just build byte_order from `np_dtype[0]` to simplify the parametrization?

Reply:
Hm, I feel like either we don't build anything dynamically or everything, i.e. keep it as is or also build `torch_dtype`. Thoughts?

Reply:
let's leave as is , it's just a nit